moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -153,7 +154,11 @@ const usePrepareActivityData = (activityData) =>
           const oldData = outputEvent.record_data.event.data.old;
           let changedField = "";
           Object.keys(newData).forEach((key) => {
-            if (newData[key] !== oldData[key]) {
+            if (!!newData[key] && typeof newData[key] === "object") {
+              if (!isEqual(newData[key], oldData[key])) {
+                changedField = key;
+              }
+            } else if (newData[key] !== oldData[key] && key !== "updated_at") {
```

Comment:
included comparing to see if there were actual differences in the fields that contain arrays. 

I also had to add this part to skip the "updated_at" column, because its always going to be different. 

I want to refactor how it this all works, but in the interest of getting the release out today, I'm leaving it like this. lemme know if this is okay

Reply:
👍 sounds good to me. it is throwing me off that this code needs to exist twice—so i guess that's what you're referring to.

Reply:
Sounds like a candidate for a spin-off issue to me! I tested the updated code, and I see the deleted rows show in the log as "Removed" 🚀